src/Styling/Home.css

```diff
@@ -128,6 +128,101 @@
   width: 44%;
 }
 
+/* Project section */
+.Project_section {
+  margin-top: 75px;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.Project_section a {
+  text-decoration: none;
+}
+
+.Project_style_1 {
```

Comment:
I’m not a huge fan of these generic styling names. I would take some time to do research on naming conventions for styles or naming in general. As your project is growing, it’s getting harder to  understand your styles. changing these to something like "Project_full" & “Project_half” makes it immensely clearer what styles belong to what ... don’t copy my example haha. come up with your own names for this exercise.